Automattic/wp-calypso
# Plans: Use 2year and 3year default interval type

Related to https://github.com/Automattic/martech/issues/3403
Branched from https://github.com/Automattic/wp-calypso/pull/96357 (term-savings logic)

## Proposed Changes

- Introduces pbxNRc-4tx-p2 that will show longer term plans by default in the plans grid. 
- Experiment variants:
  - Control: Current pricing page, defaulted to annual plans
  - Treatment 1: Savings emphasized, grid defaults to annual plans
  - Treatment 2: Savings emphasized, grid defaults to 2-year plans
  - Treatment 3: Savings emphasized, grid defaults to 3-year plans
- Adds a new experiment hook [use-longer-plan-term-default-experiment.ts](https://github.com/Automattic/wp-calypso/pull/96174/files#diff-1785e087610df8a5e98cf6fd6a8ca8adb85326f101018d83d00f8ef1d8cb5c9b)
- See https://github.com/Automattic/wp-calypso/pull/96357 for the term-savings logic (this PR builds on top to hook the experiment and add the default term)

### Media

<img width="900" alt="Screenshot 2024-11-20 at 5 14 37 PM" src="https://github.com/user-attachments/assets/bffb6f83-47fb-4ce4-b212-3e8267c14e4a">


## Why are these changes being made?


Low-tier, Personal and Premium Plan customers represent, by volume, the majority of dotcom plan purchases. With the Personal Plan alone, 45-50% of all users purchase said plan at signup. When considering monthly, yearly, 2 year, and 3 year billing terms, 35% of all users purchase an annual Personal Plan at signup.

Our low-tier users, however, are extremely price sensitive.

One potential way to address price sensitivity is to make our 2 year and 3 year plan discounts more prominent. By showing longer term plans on page load ( Ex. 2 year billing ), the plans grid monthly costs will render with 20% lower prices than with annual billing.

See pau2Xa-6wD-p2 for more context 

## Testing Instructions



* Switch to USD and go to `/start/plans` and `/plans/:site`. Confirm:
  * For **treatment** variations:
    * Treatment 1: plans grid defaults to annual plans (and term in toggle) with savings emphasized on term yearly+
    * Treatment 2: plans grid defaults to 2-year plans (and term in toggle) with savings emphasized on term yearly+ 
    * Treatment 3: plans grid defaults to 3-year plans (and term in toggle) with savings emphasized on term yearly+
  * For **control** variant:
    * plans grid defaults to annual plans (and term in toggle) with no savings emphasized  

## Pre-merge Checklist
